pages/privacy-and-permissions.tsx

```diff
@@ -0,0 +1,265 @@
+import Hero from "../components/Hero";
+import { useTranslations } from "next-intl";
+
+export async function getStaticProps({ locale }: { locale: any }) {
+  return {
+    props: {
+      metas: {
+        title:
+          "Privacy and Permissions policies, Open Source Day 2023 - Florence",
+        description:
+          "Open Source Day 2023 coming on the 24th of March 2023. Stay tuned on our social",
+      },
+      messages: (await import(`../public/locales/${locale}.json`)).default,
+    },
+  };
+}
+
+export default function CodeOfConduct() {
+  const t = useTranslations("PrivacyAndPermissions");
+  return (
+    <>
+      <div className="container">
+        <Hero title={t("title")} subtitle={t("subtitle")} originals={false} />
+        <div className="codeconduct">
+          <h3>Oggetto del trattamento</h3>
```

Comment:
We should specify either that the text is available only in italian, which is not good, or to add the text in the i18n. What do you think